src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractEncProject_IVsENCRebuildableProjectCfg.cs

```diff
@@ -20,57 +19,58 @@ public int HasCustomMetadataEmitter(out bool value)
 
         public int StartDebuggingPE()
         {
-            return EditAndContinueImplOpt.StartDebuggingPE();
+            return EditAndContinueImplOpt != null ? EditAndContinueImplOpt.StartDebuggingPE() : VSConstants.S_OK;
         }
 
         public int StopDebuggingPE()
         {
-            return EditAndContinueImplOpt.StopDebuggingPE();
+            return EditAndContinueImplOpt != null ? EditAndContinueImplOpt.StopDebuggingPE() : VSConstants.S_OK;
```

Comment:
caller might care it being failed but probably won't care if it succeeded even if we didn't actually do anything at shutdown.